### ISSUE TITLE
No way to refresh state, and destroy fails if it already doesn't exist

### ISSUE THREAD

**Issue by alex-hunt-materialize** [benesch/pulumi-postgresql-exec#1]
If an object doesn't exist (ie: it got deleted outside pulumi), there is no way to destroy it or recreate it without manually removing it from the state.

Perhaps adding a way to refresh the state of the object would fix this.

**Comment by benesch** [comment]
Yeah, like a refresh_sql command that returns whether the object still exists? That makes sense. Though am hopeful I can remove use of this provider from our cloud repo once Pulumi ships the latest version of the Terraform Postgres provider, which has native support for managing replication slots.

**Comment by alex-hunt-materialize** [comment]
Yeah, exactly like that.
